Creative/apps/creative-tv/package.json

```diff
@@ -46,7 +47,8 @@
     "ui": "*",
     "usehooks-ts": "^2.9.1",
     "wagmi": "^0.10.0",
-    "webpack": "^5.88.1"
+    "walletconnect": "^1.7.8",
```

Comment:
Do we need this installed? 
We should be importing `walletConnect` from thirdweb-dev/react